### PR TITLE
Refine submission view

### PR DESCRIPTION
**Pull request by bgrawi**

### CODE REVIEW

File: src/js/modules/submission/components/geneList/subcomponents/singleGene/singleGene.jsx [hunk 1]
```diff
@@ -42,21 +32,23 @@ class SingleGene extends React.Component {
...
-                                </InputGroupAddon>
-                                {locusNameInput}
-                                <InputGroupAddon>
-                                    <ValidationStatus validating={this.props.validating} finalized={this.props.finalized || !!this.props.validationError} isValid={this.props.validationError == ""} />
-                                </InputGroupAddon>
-                            </InputGroup>
+                            <ValidationInput
+                                title={this.props.title}
+                                value={this.props.locusNameValue}
+                                onChange={this.props.onLocusNameChange}
```
Comment (bgrawi) [inline src/js/modules/submission/components/geneList/subcomponents/singleGene/singleGene.jsx]: remove

File: src/js/modules/submission/components/geneList/subcomponents/singleGene/singleGene.jsx [hunk 2]
```diff
@@ -100,7 +92,7 @@ SingleGene.propTypes = {
     onLocusNameChange: React.PropTypes.func,
     onGeneSymbolChange: React.PropTypes.func,
     onFullNameChange: React.PropTypes.func,
-    onBlur: React.PropTypes.func,
+    validateGeneData: React.PropTypes.func,
     onKeyDown: React.PropTypes.func,
```
Comment (bgrawi) [inline src/js/modules/submission/components/geneList/subcomponents/singleGene/singleGene.jsx]: remove

File: src/js/modules/submission/components/validationInput/validationInput.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    }
+}
+
+ValidationInput.propTypes = {
+    title: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    value: React.PropTypes.string,
+    onChange: React.PropTypes.func,
+    attemptValidate: React.PropTypes.func,
+
```
Comment (bgrawi) [inline src/js/modules/submission/components/validationInput/validationInput.jsx]: nit: remove empty line

File: src/js/modules/submission/components/annotationList/subcomponents/annotationFormats/geneTerm.jsx [hunk 4]
```diff
@@ -9,6 +9,7 @@ import { annotationTypeData } from "../../../../constants";
 import KeywordTextInput from '../../../keywordTextInput';
 import CustomTextInput from "lib/components/customTextInput";
 import ValidationStatus from "../../../validationStatus";
```
Comment (GavinNishizawa) [inline src/js/modules/submission/components/annotationList/subcomponents/annotationFormats/geneTerm.jsx]: Remove ValidationStatus import

File: src/js/modules/submission/components/geneList/subcomponents/singleGene/index.js [hunk 5]
```diff
@@ -93,10 +75,9 @@ class SingleGeneContainer extends React.Component {
             onLocusNameChange: this.handleLocusNameChange,
             onGeneSymbolChange: this.handleGeneSymbolChange,
             onFullNameChange: this.handleFullNameChange,
-            onBlur: this.attemptToFinalize,
+            validateGeneData: this.props.validateGeneData,
             onKeyDown: this.handleKeyPress,
```
Comment (GavinNishizawa) [inline src/js/modules/submission/components/geneList/subcomponents/singleGene/index.js]: Remove reference to handleKeyPress

File: src/js/modules/submission/components/submissionView/submissionView.jsx [hunk 6]
```diff
@@ -9,6 +9,7 @@ import {Alert, Card, CardImg, CardText, CardHeader,
     CardBody, CardFooter, CardTitle, CardSubtitle, Button,
     ListGroup, ListGroupItem, ListGroupItemHeading,
     Form, FormGroup, Label, Input,
+    Nav, NavItem, NavLink,
```
Comment (GavinNishizawa) [inline src/js/modules/submission/components/submissionView/submissionView.jsx]: Remove unused imports

File: src/js/modules/submission/components/submissionView/submissionView.jsx [hunk 7]
```diff
@@ -43,6 +40,9 @@ class SubmissionView extends React.Component {
             </Alert>
         );
 
+        const showGenes = false;
```
Comment (GavinNishizawa) [inline src/js/modules/submission/components/submissionView/submissionView.jsx]: Remove unused variables

File: src/js/modules/submission/components/validationStatus/validationStatus.jsx [hunk 8]
```diff
@@ -21,31 +25,42 @@ class ValidationStatus extends React.Component {
             return className + "fa-spin fa-refresh";
         }
         if (this.props.finalized) {
-            return className +  (this.props.isValid? "fa-check": "fa-exclamation-circle");
+            return className +  (!this.props.validationError? "fa-check": "fa-exclamation-circle");
         } else {
             return className + "fa-chain";
         }
     }
+    toggle() {
```
Comment (GavinNishizawa) [inline src/js/modules/submission/components/validationStatus/validationStatus.jsx]: Remove toggle?

**Pull request merged**